exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/flatten/FlattenRecordBatch.java

```diff
@@ -99,6 +100,22 @@ private void clear() {
     }
   }
 
+  public enum Metric implements MetricDef {
+    NUM_INCOMING_BATCHES,
+    AVG_INPUT_BATCH_SIZE,
+    AVG_INPUT_ROW_WIDTH,
```

Comment:
`AVG_INPUT_ROW_WIDTH` to be parallel with `AVG_OUTPUT_ROW_WIDTH`?

Reply:
@paul-rogers Paul, I did not understand what you mean by parallel here and below. Do you mean they should be adjacent columns in the web UI ?

Reply:
Parallel here simply means the same name: "INCOMING"/"INPUT" --> "INPUT", "OUTGOING"/"OUTPUT" --> "OUTPUT". Same name in each term.